pkg/syncer/mutator/jsonpatch.go

```diff
@@ -26,7 +26,7 @@ func (m *JSONPatch) GetName() string {
 func (m *JSONPatch) Mutate(cfg syncer.MutatorConfig, obj *unstructured.Unstructured) error {
 	patchString := metadata.GetAnnotation(obj, JSONPatchAnnotationPrefix+cfg.ClusterID)
 	if patchString == "" {
-		return fmt.Errorf("no patch found for sync target '%v' on object: %v/%v", cfg.ClusterID, obj.GetNamespace(), obj.GetName())
+		return nil
```

Comment:
What's the reason for removing the error returned here?

Reply:
When writing the unit tests, it felt unnatural for the mutator to error just because no patches were specified, as it seems valid that something might need to be synced without being patched.

Reply:
That sounds reasonable.
What's the value if you want to just sync the resource to a cluster? An empty string `""` ?